Website/routes/admin.js

```diff
@@ -90,21 +90,45 @@ async function processUpload(req,res,next){
 
 router.post('/uploadAccount', xmlparser({trim: false, explicitArray: false,strict:false,async:true}), 
 processUpload);
-router.get('/logs', async function(req, res, next) {
+
+router.get('/access-logs', async function(req, res, next) {
     if ((req.session.secure) && (req.session.uname != "admin")) {
         return res.redirect('/home');
     }
     res.sendFile('logs/access.log', { root: '.' })
 });
 
+router.get('/deposit-logs', async function(req, res, next) {
```

Comment:
We may not need to have an individual log for each. Why not have one log called actions-log and all write to it? is that what access-logs does?

Reply:
Honestly the only reason they each have their own log file is so that when someone follows the instructions it will be very easy to find the lines they're looking for without having to sort through other stuff. I don't know if that's a great reason.

Reply:
Yeah, that's a fine reason. It does add some unnecessary code complexity but i don't think we'll be touching those logs again so it doesn't really matter.

Reply:
Yeah, and it also crossed my mind that in a properly scaled application there'd probably be more than one source of logging rather than a mega file... I think there's arguments for either way, but neither stood out to me as obviously more correct here.